Resources/Prototypes/Entities/Structures/Decoration/crystals.yml

```diff
@@ -44,7 +44,7 @@
       thresholds:
         - trigger:
             !type:DamageTrigger
-            damage: 50
+            damage: 20
```

Comment:
maybe instead of reducing health add them the ability to instantly dig with a pickaxe?

Reply:
i think they need less health anyway